capture-sdk/screen-api-example-app/src/main/java/net/gini/android/capture/screen/ui/MainActivity.kt

```diff
@@ -261,31 +246,13 @@ class MainActivity : AppCompatActivity() {
             }
         }
         Toast.makeText(
-            this, "Requirements not fulfilled:\n$stringBuilder",
-            Toast.LENGTH_LONG
+            this, "Requirements not fulfilled:\n$stringBuilder", Toast.LENGTH_LONG
         ).show()
     }
 
 
-    private fun getOnboardingPages(
```

Comment:
[Request] These onboarding pages should be shown when  "Custom onboarding pages" is enabled.